flixel/FlxG.hx

```diff
@@ -113,6 +113,11 @@ class FlxG
 	 */
 	public static var drawFramerate(default, set):Int;
 	
+	public static var renderMethod(default, null):FlxRenderMethod;
```

Comment:
We might add more render methods in the future (e.g. DRAW_TRIANGLES), and it's possible some of those might act as subsets of one of the other two. This seems to be how DRAW_TRIANGLES was set up under the old constants -- *both* DRAW_TILES and DRAW_TRIANGLES compiler constants would be set in the draw_triangles render path, and draw_triangles shared a lot of logic with draw_tiles, just with a few additional logic blocks triggered by it's own constant.